openfl-tutorials/interactive_api/HPU/PyTorch_Kvasir_UNet/README.md

```diff
@@ -4,21 +4,71 @@
 #### The name of the file/example that contain HPU adaptations start with "HPU". 
 For example: PyTorch_Kvasir_UNet.ipynb placed under workspace folder contains the required HPU adaptations.
 
- All the execution steps mention in last section (**V. How to run this tutorial**) remain same for HPU examples but as pre-requisite it needs some additional environment setup and Habana supported package installations which is explained below from **section I to IV**.
+ All the execution steps mention in last section (**V. How to run this tutorial**) remain same for HPU examples but as pre-requisite it needs some additional environment setup and Habana supported package installations which is explained below from **section I to V**.
 
  **Note:** By default these experiments utilize 1 HPU device
 
  <br/>
 
- ## **I. AWS DL1 Instance Setup**
+ ## **I. Intel Developer Cloud Setup**
+This example was test on the Intel Developer Cloud utilizing Gaudi2 instance. 
+
+For accessing the Gaudi2 instances on the Intel Developer Cloud follow the instructions [here](https://developer.habana.ai/intel-developer-cloud/)
+
+The Gaudi instance in the Intel Developer Cloud comes SynapseAI SW Stack for Gaudi2 installed. Skip sections (**II. , III.***)
+
+Further more our testing was done using the habana based Docker container built using the following Dockerfile base:
+
+```
+
+FROM vault.habana.ai/gaudi-docker/1.10.0/ubuntu20.04/habanalabs/pytorch-installer-2.0.1/latest
+
+ENV HABANA_VISIBLE_DEVICES=all
+ENV OMPI_MCA_btl_vader_single_copy_mechanism=none
+
+```
+
+This base container comes with HPU Pytorch packages already installed.  Hence you could skip step: **IV.** below. 
+
+Build the above container and then launch it using:
```

Comment:
It would help to include both a recommendation for the Dockerfile name (i.e. Dockerfile_Habana, and the docker build command here. That way these commands can be copied to terminal and run as is, and you can replace Container_Image in the docker run command with the specific image name

Reply:
Done